### PR TITLE
Refactor the `<Stack/>` component with new props

### PR DESCRIPTION
**Pull request by cmarin001**
In this PR, the `<Stack/>` component has been refactored to accommodate new prop inclusions. These changes aim to improve flexibility and increase customization options for the component, aligning with our aim to consistently enhance user experience and versatility.

### CODE REVIEW

File: src/components/layouts/Stack/stories/Stack.Default.stories.tsx [hunk 1]
```diff
@@ -24,32 +26,32 @@ const story = {
...
-    children: [...Array(6 + 1).keys()].slice(1),
-    gap: "10px",
-  },
-  render: (args: IStackProps) => <StackTemplate {...args} />,
+Default.args = {
+  children: [...Array(6 + 1).keys()].slice(1),
+  gap: "10px",
+};
+
+Default.argTypes = {
```
Comment (Andresbl123) [inline src/components/layouts/Stack/stories/Stack.Default.stories.tsx:44]: Esto ya no debería estar, para dar cumplimiento al criterio de aceptación: **The argTypes are defined in the story object and use the spread of the props object.**

File: src/components/layouts/Stack/stories/props.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
-import { flexAlignments } from "../types/Stack.FlexAlignments";
-import { directionAlignments } from "../types/Stack.DirectionAlignments";
+import { directionAlignments } from "../types/Stack.DirectionAlignment.type";
 import { wrapControls } from "../types/Stack.WrapControl.type";
+import { justifyContentProperties } from "../types/Stack.JustifyContent.type";
+import { alignItemsProperties } from "../types/Stack.AlignItems.type";
+import { alignContentProperties } from "../types/Stack.AlignContent.type";
 
 const children = {
```
Comment (Andresbl123) [inline src/components/layouts/Stack/stories/props.ts:7]: todos los argTypes, deben estar dentro de un solo objeto llamado `props`, para dar cumplimiento al criterio de aceptación: All the props documentation is exported using just the props object.

File: src/components/layouts/Stack/stories/Stack.Default.stories.tsx (hunk 1 above)
Comment (cmarin001) [reply src/components/layouts/Stack/stories/Stack.Default.stories.tsx:31]: resuelto

File: src/components/layouts/Stack/stories/props.ts (hunk 2 above)
Comment (cmarin001) [reply src/components/layouts/Stack/stories/props.ts]: resuelto

File: src/components/layouts/Stack/stories/Stack.Default.stories.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/layouts/Stack/stories/Stack.Default.stories.tsx:31]: el objeto `props` exportado debe ir dentro del objeto `story`

Comment (cmarin001) [reply src/components/layouts/Stack/stories/Stack.Default.stories.tsx]: resuelto

**Review by Andresbl123 — APPROVED** [review]

**Pull request merged**